crates/abq_cli/src/main.rs

```diff
@@ -230,14 +232,18 @@ fn get_inferred_run_id(run_id_environment: RunIdEnvironment) -> Option<RunId> {
         buildkite_build_id,
         circle_workflow_id,
         github_run_id,
+        mint_run_id,
     } = run_id_environment;
 
     if abq_run_id.is_ok() || ci.unwrap_or_else(|_| String::from("false")) == *"false" {
         return None;
     }
     // note: if you change this, change it in setup-abq, too
     // https://github.com/rwx-research/setup-abq/blob/8c5044343fceef53b4efea83d64062f006bf6758/src/index.ts#L23-L28
```

Comment:
Sorry that you ran into this outdated comment.

Reply:
Oh I probably should remove this